docs/farming-&-staking/farming/space-acres/space-acres-install.mdx

```diff
@@ -0,0 +1,107 @@
+    - Substrate Node P2P Port
+    - Subspace DSN P2P Port
+    - Enable Faster Networking (QUIC)
+
+    For more information on networking and its implications, refer to our [networking guide.](../additional-guides/networking)
+
+    ![space-acres-setup-3](/img/doc-imgs/space-acres/space-acres-setup-3.png)
+
+4. The application will then proceed to load, creating the consensus node and connecting to the network.
+
+    ![space-acres-setup-4](/img/doc-imgs/space-acres/space-acres-setup-4.png)
+
+5. Finally, you will see the main interface of Space Acres, where your consensus node is running and your farmer is progressing along. Initial syncing & plotting may take some time to complete. Below are examples of the various stages you can expect. 
+
+    a. Cache Syncing
+
+    ![space-acres-setup-5](/img/doc-imgs/space-acres/space-acres-setup-5.png) 
+
+    b. Initial Plotting
+
+    ![space-acres-setup-6](/img/doc-imgs/space-acres/space-acres-setup-6.png)
+
+    c. Plotting Complete, now farming.
+
+    ![space-acres-setup-6](/img/doc-imgs/space-acres/space-acres-setup-7.png)
+
+6. With Space Acres configured, your farming operation will commence seamlessly. Keep an eye on the application to track the progress of your farm and the accumulation of rewards. For your convenience, we've provided a table detailing the color codes associated with different sector statuses. This will enhance your insight into the activities and conditions of your plot.
+
+    | Status            | Color Preview | Description                          |
+    |-------------------|---------------|--------------------------------------|
+    | Default           | ![#727272](https://via.placeholder.com/15/727272/000000?text=+) #727272 | Default state of a farm sector       |
+    | Plotted           | ![#9fc4ff](https://via.placeholder.com/15/9fc4ff/000000?text=+) #9fc4ff | Sector is plotted                    |
+    | About to Expire   | ![#ffd678](https://via.placeholder.com/15/ffd678/000000?text=+) #ffd678 | Sector is nearing expiration         |
+    | Expired           | ![#ff877c](https://via.placeholder.com/15/ff877c/000000?text=+) #ff877c | Sector has expired                   |
+    | Downloading       | ![#499249](https://via.placeholder.com/15/499249/000000?text=+) #499249 | Sector is in the process of downloading (animated) |
+    | Encoding          | ![#ff5365](https://via.placeholder.com/15/ff5365/000000?text=+) #ff5365 | Sector is encoding data (animated)   |
+    | Writing           | ![#9fc4ff](https://via.placeholder.com/15/9fc4ff/000000?text=+) #9fc4ff | Sector is writing data (animated)    |
+
+
+
+:::note Learn More About Syncing, Plotting & Farming
+
+If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Subnomicon links below.
+- [Synchronization](https://subnomicon.subspace.network/docs/network/network_protocols/#synchronization)
```

Comment:
Same comment about `-` here. Works fine as is, might let us down in the future.

Reply:
Fixed in https://github.com/subspace/subspace-docs/pull/448